lib/task/import/validate/csvColumnNameValidator.class.php

```diff
@@ -65,26 +65,31 @@ public function testRow(array $header, array $row)
         }
 
         foreach ($header as $columnName) {
-            // If $columnName is not in validColumnNames.
-            if (!in_array($columnName, $this->validColumnNames)) {
-                // Test for leading or trailing whitespace.
-                if (in_array(trim($columnName), $this->validColumnNames)) {
-                    $this->trimIssuesColumnNames[$columnName] = trim($columnName);
-                }
-                // Test if a match is found using lowercase trimmed matching.
-                elseif (false !== $key = array_search(strtolower(trim($columnName)), $this->validColumnNamesLowercase)) {
-                    // Map unknown column name to possible match.
-                    $this->caseIssuesColumnNameMap[trim($columnName)] = $this->validColumnNames[$key];
-                }
-
-                // Add to unknown column name list.
-                $this->unknownColumnNames[$columnName] = $columnName;
-            }
+            $this->testColumnName($columnName);
         }
 
         $this->complete = true;
     }
 
+    protected function testColumnName(string $columnName)
+    {
+        // If $columnName is not in validColumnNames.
+        if (!in_array($columnName, $this->validColumnNames)) {
```

Comment:
I'd avoid nested if statements with a return:

```
if (in_array($columnName, $this->validColumnNames)) {
    return;
}
```

Reply:
This change was already made. csvColumnNameValidator::testColumnName() no longer has this return statement.